GoogleForCreators/web-stories-wp
# Media: Show duration while uploading files

## Context



## Summary

While uploading a file from local using drag and drop, it should be possible to get length / duration of the video to preview.

## Relevant Technical Choices



## To-do



## User-facing changes

### Before

<img width="344" alt="Screenshot 2021-05-24 at 14 05 01" src="https://user-images.githubusercontent.com/237508/119352330-73938d00-bc99-11eb-9234-8f11633da3b4.png">

### After 

<img width="358" alt="Screenshot 2021-05-24 at 13 59 35" src="https://user-images.githubusercontent.com/237508/119352364-7ee6b880-bc99-11eb-80a7-38ca99bccba3.png">


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Open the editor. 
2. Drag video into canvas. 
3. See duration of video while uploading. 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7693
